code/datums/saymode.dm

```diff
@@ -0,0 +1,112 @@
+GLOBAL_LIST_EMPTY(saymodes)
+
+/datum/saymode
+	var/key
+	var/mode
+
+/datum/saymode/proc/handle_message(mob/living/user, message, datum/language/language)
+	return FALSE
```

Comment:
I'd invert the logic here, so things only return TRUE if they weren't handled. Then you could eliminate all the return TRUEs in the overrides